### ISSUE TITLE
Add Functioning Demo

### ISSUE THREAD

**Issue by jermsull** [hyzhak/ui-select-infinity#4]
Is it possible to add a demo of ui-select-infinity so that people can see how you recommend using ui-select with the infinity scrolling functionality?

**Comment by hyzhak** [comment]
@jermsull good point. Is example from README not enough? https://github.com/hyzhak/ui-select-infinity#example

**Comment by umdstu** [comment]
Would be useful to see a server side implementation called in getInfinityScrollChunk(id); For example, what the request would look like and the response would look like, as compared to angular-ui-select.

Update: esp. considering I haven't been able to get it to work yet ;)

**Comment by jermsull** [comment]
Thanks for the quick response, Eugene. I'm pretty new to AngularJS, and it would be nice to see how to combine ui-select, ui-select-infinity, and a data source with infinite scrolling.

**Comment by umdstu** [comment]
For example, whats the connection between refreshList() and getInfinityScrollChunk(id) ?

**Comment by hyzhak** [comment]
ok, I see, it will take significant amount of time. So will put it in my TODO list but can't give you example right away. But I always open for PR, so if you will find solution please send commits with it

**Comment by andrewboni** [comment]
+1 for this, would be very helpful

**Comment by varunoberoi** [comment]
+1 for a plunker

**Comment by hyzhak** [comment]
@andrewboni @varunoberoi ok, are there any ideas which public API with pagination should we use as a demo?

**Comment by varunoberoi** [comment]
Just replicate original select2's infinite scroll demo using following API: 
https://developer.github.com/v3/search/#search-repositories

Demo:
http://select2.github.io/select2/#infinite

**Comment by guilhermecvmnsj** [comment]
Example is not working, any help?

**Comment by vhalthi** [comment]
An example would be very helpful, Is there any working example (demo) ?

**Comment by hyzhak** [comment]
@1HzAkLAT sadly I'm very occupied right now. But I'm opened for PR contributions

**Comment by sharonx** [comment]
Here is the example: http://plnkr.co/edit/rzxHXGUm9eKEUqRSKWyj?p=preview before my PR is merged

**Comment by tonidy** [comment]
@sharonx it seems your plunker not working

**Comment by sharonx** [comment]
It works for me, at least on chrome. I haven’t tested it anywhere else. 

Can you give me more details about how it’s not working?

-sharonx

**Comment by tonidy** [comment]
@sharonx sorry it is working now, I have issue with my Internet connection

**Comment by pencilcheck** [comment]
There seems to be a bug in the inifite-scrolling module logic when it initializes. I fixed it now it is working for me. Here is an example: http://plnkr.co/edit/p6Dt3yvTq40Vn56AKiqC?p=preview

**Comment by hyzhak** [comment]
@pencilcheck ok, I see. You can create pull request with your changes. So I could merge them to repository

**Comment by sergiogregorutti** [comment]
@hyzhak thanks for this extension. It is really useful.

@pencilcheck can you create a pull request with your changes? It will be very appreciated. Thanks!

**Comment by pencilcheck** [comment]
Yup. https://github.com/hyzhak/ui-select-infinity/pull/12
